packages/attention/src/component.tsx

```diff
@@ -137,54 +141,58 @@ export function Attention(props: AttentionProps) {
   useAutoUpdatePosition(targetEl, isShowing, attentionEl, autoUpdatePosition, attentionState)
   
   return (
-    <div
-      data-testid='attention-el'
-      className={classNames(
-        {
-          [ccAttention.notCallout]: !props.callout,
-          invisible: !isVisible && !props.callout,
-          hidden: !isVisible && !props.tooltip,
-        },
-        className
-      )}
-      ref={attentionEl}
-    >
-      <div
-        role={props.role === '' ? undefined : props.tooltip ? 'tooltip' : 'img'}
-        aria-label={
-          ariaLabel === '' ? undefined : ariaLabel ?? defaultAriaLabel()
-        }
-        className={wrapperClasses}
-        id={props.id}
-      >
-        {!props.noArrow && (
-          <Arrow {...props} ref={arrowEl} direction={actualDirection} />
-        )}
-        <div className={ccAttention.content}>{props.children}</div>
-        {canClose && (
-          <button
-            type='button'
-            aria-label={i18n._(
-              /*i18n*/ {
-                id: 'attention.aria.close',
-                message: 'Close',
-                comment: 'Aria label for the close button in attention',
-              }
-            )}
-            onClick={onDismiss}
-            onKeyDown={(event) => {
-              if (!props.onDismiss) return
-              if (event.key === 'Escape') {
-                props.onDismiss()
-              }
-            }}
-            className={ccAttention.closeBtn}
+    <>
+      {(props.callout || (props.targetEl !== undefined && !props.callout)) && (
```

Comment:
What do you think about doing the opposite here, too, and returning `null` if `!props.callout && props.targetEl === undefined`?

Reply:
Yes, I was thinking of doing the same here as well! 🎉 However, not sure if I should make the same changes for the vue attention component: https://github.com/warp-ds/vue/pull/161/files#diff-73fbe20fc6b40d2120b1a5e414f088ed51cea513e1eb9745402ee02a639b22b0R220

Reply:
Yeah, sounds like Vue should stay as is with the v-if syntax.

Reply:
Updated in this commit: [5c799d2](https://github.com/warp-ds/react/pull/233/commits/5c799d27f0224b55b99cfc94b1f81b924506a6d8)